styles.css

```diff
@@ -1131,16 +1131,56 @@ header {
   justify-content: center;
 }
 
+/*.subscribe__form form {
+  width: 100%;
+  max-width: 400px;
+  display: flex;
+  background-color: var(--extra-light);
+  box-shadow: 3px 3px 30px blue;
+  border-radius: 5rem;
+  overflow: hidden;
+  border: 0.001px solid gray;
+}*/
+
 .subscribe__form form {
   width: 100%;
   max-width: 400px;
   display: flex;
+  align-items: center;
   background-color: var(--extra-light);
   box-shadow: 3px 3px 30px slateblue;
   border-radius: 5rem;
   overflow: hidden;
   border: 0.001px solid gray;
 }
+.subscribe__form input[type="email"] {

```

Comment:
good work . thanks for your valuable contribution

Reply:
Thank you Sir